src/node_crypto.cc

```diff
@@ -2157,11 +2160,6 @@ void SSLWrap<Base>::SetNPNProtocols(const FunctionCallbackInfo<Value>& args) {
 #endif  // OPENSSL_NPN_NEGOTIATED
 
 #ifdef TLSEXT_TYPE_application_layer_protocol_negotiation
-typedef struct tlsextalpnctx_st {
-  unsigned char* data;
-  unsigned short len;
-} tlsextalpnctx;
```

Comment:
The linter was complaining about the `unsigned short`.  I figured if it's unused I might as well remove it instead of adding a NOLINT.